testplan/report/testing/schemas.py

```diff
@@ -113,9 +114,10 @@ class TestGroupReportSchema(TestCaseReportSchema):
     category = fields.String()
     part = fields.List(fields.Integer, allow_none=True)
     fix_spec_path = fields.String(allow_none=True)
-    env_status = fields.String(allow_none=True)
+    # env_status = fields.String(allow_none=True)
```

Comment:
Why is this removed...? Please uncomment.

Reply:
done